lib/editor/index.js

```diff
@@ -125,27 +125,34 @@ export default class Editor {
   }
   listenForMouseMovement() {
     const editorElement = atom.views.getView(this.textEditor)
-    return disposableEvent(editorElement, 'mousemove', debounce((e) => {
-      if (!editorElement.component || !hasParent(e.target, 'div.scroll-view')) {
+
+    return disposableEvent(editorElement, 'mousemove', debounce((event) => {
+      if (!editorElement.component || !hasParent(event.target, 'div.scroll-view')) {
         return
       }
       const tooltip = this.tooltip
-      if (tooltip && mouseEventNearPosition(e, editorElement, tooltip.marker.getStartScreenPosition(), tooltip.element.offsetWidth, tooltip.element.offsetHeight)) {
+      if (tooltip && mouseEventNearPosition({
+        event,
+        editor: this.textEditor,
+        editorElement,
+        tooltipElement: tooltip.element,
+        screenPosition: tooltip.marker.getStartScreenPosition(),
+      })) {
         return
       }
-      // NOTE: Ignore if file is too big
+
+      let cursorPosition = getBufferPositionFromMouseEvent(event, this.textEditor, editorElement)
```

Comment:
Is there a reason you don't just update `this.cursorPosition` directly here? The rest of the code seems to switch randomly between which it uses, you might as well just use it directly as there is no reason not to.